test/cases/aws.sh

```diff
@@ -258,13 +258,15 @@ cp "${CIV_CONFIG_FILE}" "${TEMPDIR}/civ_config.yml"
 # temporary workaround for
 # https://issues.redhat.com/browse/CLOUDX-488
 if nvrGreaterOrEqual "osbuild-composer" "83"; then
+    # TODO: remove test filtering using "-t" once https://issues.redhat.com/browse/RHEL-24346 is resolved
     sudo "${CONTAINER_RUNTIME}" run \
         -a stdout -a stderr \
         -e AWS_ACCESS_KEY_ID="${V2_AWS_ACCESS_KEY_ID}" \
         -e AWS_SECRET_ACCESS_KEY="${V2_AWS_SECRET_ACCESS_KEY}" \
         -e AWS_REGION="${AWS_REGION}" \
         -e JIRA_PAT="${JIRA_PAT}" \
         -v "${TEMPDIR}":/tmp:Z \
+        -t "not (TestsGeneric and test_no_avc_denials)" \
```

Comment:
That's one way of dealing with it although not my preferred one.

Do you mind opening a ticket for us to revert this change once the blocker is fixed? 

Otherwise LGTM.

Reply:
I agree. As commented [here](https://github.com/osbuild/osbuild-composer/pull/4003#issuecomment-1990419242) I think the best is for us (cloudx team) to add the jira_skip marker with the corresponding bug in JIRA.